usr.sbin/sesutil/eltsub.c

```diff
@@ -12,7 +12,7 @@
  *    derived from this software without specific prior written permission.
  *
  * Alternatively, this software may be distributed under the terms of the
- * the GNU Public License ("GPL").
+ * GNU Public License ("GPL").
```

Comment:
We'd need Matthew Jacob's permission for these...  And I'm not sure how that would go.